src/routes/userRoute.ts

```diff
@@ -0,0 +1,160 @@
+import { Hono } from 'hono';
+import { PrismaClient } from '@prisma/client';
+import { validator } from 'hono/validator';
+import {
+  schemaEmail,
+  schemaId,
+  schemaLogin,
+  schemaPassword,
+  schemaUser,
+} from '../models/userSchema';
+import bcrypt from 'bcrypt';
+
+const prisma = new PrismaClient();
+
+export const userRoute = new Hono();
+
+userRoute.get('/', async (c) => {
+  return c.text('Hello!');
+});
+
+userRoute.get('/users', async (c) => {
+  const allUsers = await prisma.user.findMany();
+  return c.json({ data: allUsers, ok: true });
+});
+
+userRoute.get(
+  '/users/:id',
+  validator('param', (value, c) => {
```

Comment:
When you use `parsedId.data` in the return and TS will start yelling at you, you can prevent it with `value: { id: string }`